Hunting Queries/BehaviorAnalytics/Anomalous Failed Logon.yaml

```diff
@@ -0,0 +1,23 @@
+id: 77a58c08-bc8e-48b3-8fe9-6c38fd011cd9
+name: Anomalous Failed Logon
+description: |
+  'Users with 'High' BlastRadius that perform failed Sign-in:Invalid username or password.'
+requiredDataConnectors:
+  - connectorId: BehaviorAnalytics
+    dataTypes:
+      - BehaviorAnalytics
+      - SigninLogs
+tactics:
+  - Credential Access
+relevantTechniques:
+  - T1110.001
+query: |
+
+ BehaviorAnalytics
+ | where ActivityType == "LogOn"
+ | where UsersInsights.BlastRadius == "High"
+ | join (
+ SigninLogs
+ ) on $left.SourceRecordId == $right._ItemId
+ | where Status.errorCode == 50126

```

Comment:
This should be a filter inside the join?

Reply:
done